dev-python/spyder/spyder-4.1.1.ebuild

```diff
@@ -33,7 +33,7 @@ RDEPEND="
 	dev-python/keyring[${PYTHON_USEDEP}]
 	>=dev-python/nbconvert-4.0[${PYTHON_USEDEP}]
 	>=dev-python/numpydoc-0.6.0[${PYTHON_USEDEP}]
-	~dev-python/parso-0.5.2[${PYTHON_USEDEP}]
+	>=dev-python/parso-0.5.2[${PYTHON_USEDEP}]
```

Comment:
This requires a revbump for everyone to get the updated requirement changes (will do it for you, just FYI)

Reply:
Alright, Thank You!